src/managing-workflow/upgrading-workflow.md

```diff
@@ -1,41 +1,34 @@
 # Upgrading Workflow
 
-To upgrade to a newer version of Workflow, run the following:
-
-```
-$ helmc uninstall workflow-beta4 -n deis
-$ helmc fetch deis/workflow-rc1
-$ helmc generate -x manifests workflow-rc1
-$ cp `helmc home`/workspace/charts/workflow-beta4/manifests/deis-database-secret-creds.yaml \
-    `helmc home`/workspace/charts/workflow-rc1/manifests/
-$ helmc install workflow-rc1
-```
+Deis Workflow releases may be upgraded in-place with minimal downtime. This upgrade process requires:
 
-Make sure to copy the existing `deis-database-secret-creds.yaml` manifest into the new chart
-location *after* `helmc generate` to keep database credentials intact.
+* Helm Classic, version 0.8.0 or newer
+* Configured Off-Cluster Storage
+* A copy of the database and builder secrets from your running cluster
+* Workflow manifests annotated with `helm-keep: true` (rc1 or later)
 
 ## Off-Cluster Storage Required
 
 A Workflow upgrade requires using off-cluster object storage, since the default
-in-cluster storage is ephemeral. Upgrading Workflow with the in-cluster default
-of [Minio][] will result in data loss.
+in-cluster storage is ephemeral. **Upgrading Workflow with the in-cluster default
+of [Minio][] will result in data loss.**
 
 See [Configuring Object Storage][] to learn how to store your Workflow data off-cluster.
 
-## Keeping Essential Pieces
+## Keeping Essential Components
 
 !!! note
```

Comment:
This whole note can probably be removed now that you are stating the requirements up above.